fgrpc/pingsrv_test.go

```diff
@@ -29,28 +30,53 @@ func init() {
 }
 
 func TestPingServer(t *testing.T) {
-	port := PingServer("0", "foo")
-	addr := fmt.Sprintf("localhost:%d", port)
-	t.Logf("test grpc ping server running, will connect to %s", addr)
-	if latency, err := PingClientCall(addr, false, 7, "test payload"); err != nil || latency <= 0 {
+	iPort := PingServer("0", "", "", "foo")
+	iAddr := fmt.Sprintf("localhost:%d", iPort)
+	t.Logf("test insecure grpc ping server running, will connect to %s", iAddr)
+	sPort := PingServer("0", "../testdata/server.crt", "../testdata/server.key",
```

Comment:
isn't this relative path making success dependent on where it is ran from ? (somehow doubt it works from both parent or different directories?)

not sure we need a new directory - tbd if go testing has provision for files relative to test files?